addon/templates/components/bs4/bs-accordion/item.hbs

```diff
@@ -0,0 +1,12 @@
+<div role="tab" {{action onClick value}} class="card-header {{if collapsed "collapsed" "expanded"}}">
+    <h5 class="card-title mb-0">
```

Comment:
`mb-0` applies `margin-bottom: 0`.  This follows the example in the Bootstrap 4 docs for their Accordion - without it the header ends up being too tall